src/ontology/mondo-edit.obo

```diff
@@ -455619,19 +455597,35 @@ subset: rare
 synonym: "buphthalmia" EXACT [Orphanet:98976]
 synonym: "buphthalmos" EXACT [Orphanet:98976]
 synonym: "Buphthalmus" EXACT [Orphanet:98976]
+synonym: "hydrophthalmos" EXACT [MONDO:0006788]
 synonym: "primary congenital glaucoma" NARROW [Orphanet:98976]
-xref: ICD10CM:Q15.0 {source="Orphanet:98976", source="Orphanet:98976/e", source="Orphanet:98976/specific"}
+xref: DOID:11212 {source="MONDO:equivalentTo", source="EFO:1000968"}
+xref: ICD10CM:Q15.0 {source="Orphanet:98976", source="Orphanet:98976/e", source="Orphanet:98976/specific", source="DOID:11212"}
+xref: ICD9:743.2 {source="DOID:11212"}
+xref: ICD9:743.20 {source="DOID:11212"}
 xref: MEDGEN:42532 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
-xref: NCIT:C50648 {source="MONDO:equivalentTo"}
+xref: MESH:D006871 {source="MONDO:equivalentTo", source="EFO:1000968", source="DOID:11212"}
+xref: NCIT:C50648 {source="MONDO:equivalentTo", source="DOID:11212"}
 xref: Orphanet:98976
```

Comment:
Can this orphanet x-ref be equivalentTo?

Reply:
yes, added